nms/app/packages/magmalte/server/app.js

```diff
@@ -93,13 +89,46 @@ app.set('views', path.join(__dirname, '..', 'views'));
 app.set('view engine', 'pug');
 
 // Routes
-app.use(
-  webpackSmartMiddleware({
-    devMode: DEV_MODE,
-    devWebpackConfig: require('../config/webpack.config.js'),
-    distPath: require('../config/paths').distPath,
-  }),
-);
+// TO DO - fix this in webpack-dev-middleware code in fbc-js-core
```

Comment:
```suggestion
// TODO - fix this in webpack-dev-middleware code in fbc-js-core
```
will be easier to find later

Reply:
Also maybe say something other than "fix this"

Reply:
Also can you pull out the stuff in the `if (DEV_MODE)`? I like keeping `app.js` as clean as possible since it's the main entrypoint